CoreScriptsRoot/ServerCoreScripts/ServerChat/ChatServiceRunner.lua

```diff
@@ -241,6 +251,9 @@ if modules then
 	end
 end
 
+wait()
+wait()
+
```

Comment:
It's to wait 2 frames to give the chat modules, which a bunch of different threads are spawned for, to run and initialize before creating players to put in channels.

I'll put a comment about it in the source file.